gazebo_plugins/src/gazebo_ros_camera.cpp

```diff
@@ -398,6 +391,9 @@ void GazeboRosCamera::Load(gazebo::sensors::SensorPtr _sensor, sdf::ElementPtr _
     auto border_crop = _sdf->Get<bool>("border_crop", true).first;
     auto hack_baseline = _sdf->Get<double>("hack_baseline", 0.0).first;
 
+    double default_cx = (static_cast<double>(width[i]) + 1.0) / 2.0;
+    double default_cy = (static_cast<double>(height[i]) + 1.0) / 2.0;
```

Comment:
Do we need to compute defaults from the image size if we're just overriding these from `LensDistortion()->Center()` below?

Reply:
yeah, so I checked the behaviour we do need to compute default `cx, cy` from image size reason being the `LensDistortion()->Center()` thing only works if their is a distortion tag provided in world file. 

https://github.com/ros-simulation/gazebo_ros_pkgs/blob/612692e3c6a00d662b5bf02461e80b928e7dc119/gazebo_plugins/src/gazebo_ros_camera.cpp#L407